<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1173</width>
    <height>594</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gerenciador de Processos Python</string>
  </property>
  <property name="styleSheet">
   <string notr="true">Qlabel: {
	font: 11pt &quot;Cantarell&quot;;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label__time">
            <property name="text">
             <string>Horário atual:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label__uptime">
            <property name="text">
             <string>Maquina ativa:</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Minimum</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label__cpu_usage">
            <property name="text">
             <string>Total de CPU utilizado:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label__memory_usage">
            <property name="text">
             <string>Total de memória utilizada:</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Minimum</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="label__infos_process">
          <property name="text">
           <string>Total de processos:</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableWidget" name="table__list_process">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QTableWidget::item:selected {
	background-color: lightgray;
	color: red;
}
QHeaderView::section {
	background-color: rgb(94, 92, 100);
	color: white;
	border: 1px solid gray;
}</string>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <property name="showGrid">
         <bool>true</bool>
        </property>
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>131</number>
        </attribute>
        <row/>
        <row/>
        <column>
         <property name="text">
          <string>PID</string>
         </property>
         <property name="font">
          <font>
           <bold>true</bold>
          </font>
         </property>
        </column>
        <column>
         <property name="text">
          <string>USER</string>
         </property>
         <property name="font">
          <font>
           <bold>true</bold>
          </font>
         </property>
        </column>
        <column>
         <property name="text">
          <string>NAME</string>
         </property>
         <property name="font">
          <font>
           <bold>true</bold>
          </font>
         </property>
        </column>
        <column>
         <property name="text">
          <string>PRIORITY</string>
         </property>
         <property name="font">
          <font>
           <bold>true</bold>
          </font>
         </property>
        </column>
        <column>
         <property name="text">
          <string>MEMORY USAGE (KB)</string>
         </property>
         <property name="font">
          <font>
           <bold>true</bold>
          </font>
         </property>
        </column>
        <column>
         <property name="text">
          <string>DISK READ (KB)</string>
         </property>
         <property name="font">
          <font>
           <bold>true</bold>
          </font>
         </property>
        </column>
        <column>
         <property name="text">
          <string>DISK WRITE (KB)</string>
         </property>
         <property name="font">
          <font>
           <bold>true</bold>
          </font>
         </property>
        </column>
        <column>
         <property name="text">
          <string>STATUS</string>
         </property>
         <property name="font">
          <font>
           <bold>true</bold>
          </font>
         </property>
        </column>
        <item row="0" column="0">
         <property name="text">
          <string>1</string>
         </property>
        </item>
        <item row="0" column="1">
         <property name="text">
          <string>root</string>
         </property>
        </item>
        <item row="0" column="2">
         <property name="text">
          <string>systemd</string>
         </property>
        </item>
        <item row="0" column="3">
         <property name="text">
          <string>-20(low)</string>
         </property>
        </item>
        <item row="0" column="4">
         <property name="text">
          <string>4123</string>
         </property>
        </item>
        <item row="0" column="5">
         <property name="text">
          <string>4312123.2</string>
         </property>
        </item>
        <item row="0" column="6">
         <property name="text">
          <string>5123.2</string>
         </property>
        </item>
        <item row="0" column="7">
         <property name="text">
          <string>I(idle)</string>
         </property>
        </item>
        <item row="1" column="0">
         <property name="text">
          <string>2</string>
         </property>
        </item>
        <item row="1" column="1">
         <property name="text">
          <string>root</string>
         </property>
        </item>
        <item row="1" column="2">
         <property name="text">
          <string>systemd</string>
         </property>
        </item>
        <item row="1" column="3">
         <property name="text">
          <string>0(default)</string>
         </property>
        </item>
        <item row="1" column="4">
         <property name="text">
          <string>30112</string>
         </property>
        </item>
        <item row="1" column="5">
         <property name="text">
          <string>6212.5</string>
         </property>
        </item>
        <item row="1" column="6">
         <property name="text">
          <string>52234.5</string>
         </property>
        </item>
        <item row="1" column="7">
         <property name="text">
          <string>S(sleeping)</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button__stop">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 190, 111);
font: 11pt &quot;Cantarell&quot;;</string>
        </property>
        <property name="text">
         <string>STOP</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button__resume">
        <property name="styleSheet">
         <string notr="true">font: 11pt &quot;Cantarell&quot;;
background-color: rgb(143, 240, 164);</string>
        </property>
        <property name="text">
         <string>RESUME</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button__kill">
        <property name="styleSheet">
         <string notr="true">font: 11pt &quot;Cantarell&quot;;
background-color: rgb(246, 97, 81);</string>
        </property>
        <property name="text">
         <string>KILL</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button__force_kill">
        <property name="styleSheet">
         <string notr="true">font: 800 11pt &quot;Cantarell&quot;;
background-color: rgb(224, 27, 36);</string>
        </property>
        <property name="text">
         <string>FORCE KILL</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
